skeleton/src/main/java/uk/gov/hmcts/${{ values.product }}/${{ values.component | replace("-", "") }}/config/OpenAPIConfiguration.java

```diff
@@ -14,13 +14,13 @@
     @Bean
     public OpenAPI openAPI() {
         return new OpenAPI()
-            .info(new Info().title("rpe demo")
-                      .description("rpe demo")
+            .info(new Info().title("${{ values.product}}.${{ values.component | replace("-", "") }}")
+                      .description("${{ values.product}}.${{ values.component | replace("-", "") }}")
                       .version("v0.0.1")
                       .license(new License().name("MIT").url("https://opensource.org/licenses/MIT")))
             .externalDocs(new ExternalDocumentation()
                               .description("README")
-                              .url("https://github.com/hmcts/template-spring-boot"));
+                              .url("https://github.com/hmcts/${{ values.product}}.${{ values.component | replace("-", "") }}"));
```

Comment:
you don't need to replace - here and you should use a - to separate the name instead of a .